draft-irtf-cfrg-frost.md

```diff
@@ -413,10 +415,11 @@ at x-coordinate 0, i.e., `f(0)`, given a list of `t` other x-coordinates.
   derive_lagrange_coefficient(x_i, L):
 
   Inputs:
-  - x_i, an x-coordinate contained in L, a Scalar
-  - L, the set of x-coordinates, each a Scalar
+  - x_i, an x-coordinate contained in L, a Scalar.
+  - L, the set of x-coordinates, each a Scalar.
 
-  Outputs: L_i, the i-th Lagrange coefficient
+  Outputs:
+  - L_i, the i-th Lagrange coefficient, a Scalar.
```

Comment:
The output name will change in https://github.com/cfrg/draft-irtf-cfrg-frost/pull/373

Reply:
Rebased to resolve conflicts!